### ISSUE TITLE
Support decompiling with line numbers

### ISSUE THREAD

**Issue by dgileadi** [dgileadi/vscode-java-decompiler#4]
Now that java debug support is improving, it may be important to decompile with line numbers.

**Comment by pmorange** [comment]
Hi,
Was it ever implemented ? Decompiler is of no much use without this feature...
Thanks.

**Comment by dgileadi** [comment]
No. I haven't had time to work on this project for a while. Pull requests are always welcome :)
